README.md

```diff
@@ -2,9 +2,100 @@
 
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-MicroCeph is snap-deployed Ceph with built-in clustering.
+<p align="center">
+<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
+</p>
 
-MicroCeph handles scaling out the monitor cluster, as well as automating placement and management of the manager and metadata processes in a Ceph cluster when units are dynamically added via easy-to-use tooling. It also helps with adding disks (OSDs) to the Ceph cluster.
+## Table of Contents
+1. [💡 Philosophy](#💡-philosophy)
+2. [🎯 Features](#🎯-features)
+3. [📖 Documentation](#📖-documentation)
+4. [⚡️ Quickstart](#⚡️-quickstart)
+5. [👍 How Can I Contribute ?](#👍-how-can-i-contribute)
 
-Documentation: https://canonical-microceph.readthedocs-hosted.com/en/latest/
-Snap: https://snapcraft.io/microceph
+## 💡 Philosophy
+
+Deploying and operating a Ceph cluster is complex, It is because Ceph is designed to be a general purpose storage solution. This is a significant overhead for smaller Ceph clusters. [MicroCeph](https://snapcraft.io/microceph) solves this by being _opinionated_ and _focused_ at small scale. With MicroCeph, deploying and operating a Ceph cluster is as easy as a [Snap!](https://snapcraft.io/microceph)
```

Comment:
It looks like the first sentence here is trying to be two different sentences; maybe it would be better as something like "Deploying and operating a Ceph cluster is complex because Ceph is designed..."

Reply:
Ack